donate/urls.py

```diff
@@ -15,8 +15,7 @@
 from donate.payments import urls as payments_urls
 from donate.payments.braintree_webhooks import BraintreeWebhookView
 from donate.payments.stripe_webhooks import StripeWebhookView
-from donate.views import EnvVariablesView, ThunderbirdRedirectView, WaysToGiveView
-
+from donate.views import EnvVariablesView, ThunderbirdRedirectView, WaysToGiveView, apple_pay_domain_association_view
 # Patterns not subject to i18n
```

Comment:
I think the empty line here go lost on accident. What do you think about adding it back?